packages/icons/src/utils-node.ts

```diff
@@ -0,0 +1,53 @@
+//
+// this file contains utilities that require Node APIs like "fs"
+// because otherwise e.g. the docs build step will fail when importing
+// the grouped icons by category because node APIs like "node:fs"
+// are not available there
+//
+import fs from "node:fs/promises";
+import path from "node:path";
+import { fileURLToPath } from "node:url";
+import { optimize } from "svgo";
+
+/**
+ * Optimizes the given SVG content and removes all fills so the color can be set via CSS.
+ */
+export const optimizeSvg = (svgContent: string, path: string) => {
+  const { data } = optimize(svgContent, {
+    path,
+    multipass: true,
+    plugins: [
```

Comment:
The description says that we reduce the file size using svgo, but here we only use `removeAttrs`, shouldn't we at least use the `default-preset`?

Reply:
I though this is included by default^^ I added it, thanks!